UNRELEASED.md

```diff
@@ -19,6 +19,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 - Prevented the `onSelect` prop of `Tabs` from changing scroll position ([#2196](https://github.com/Shopify/polaris-react/pull/2196))
 - Fixed 200ms visual delay when activating `Popover` ([#2209](https://github.com/Shopify/polaris-react/pull/2209))
 - Fixed `ResourceList` `Item` deselect ([#1952](https://github.com/Shopify/polaris-react/pull/1952))
+- Fixed SubHeading’s `font-weight` ([#2218](https://github.com/Shopify/polaris-react/pull/2218))
```

Comment:
```suggestion
- Fixed `Subheading`’s `font-weight` ([#2218](https://github.com/Shopify/polaris-react/pull/2218))
```

Code style for the component name? And looks like it is treated as a single word: https://polaris.shopify.com/components/titles-and-text/subheading

Reply:
Oops, my bad! Good eye